src/reason_pprint_ast.ml

```diff
@@ -4017,11 +4094,21 @@ class printer  ()= object(self:'self)
   method expression x =
     let (arityAttrs, docAtrs, stdAttrs) = partitionAttributes x.pexp_attributes in
```

Comment:
I think this all might be cleaner if `partitionAttributes` also returned any `jsx` attributes it found. I've thought of `stdAttrs` as anything printed as `[@]`, but in the diff's current form, JSX gets added to `stdAttrs`.